src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx

```diff
@@ -0,0 +1,165 @@
+  - /docs/understand-dependencies/understand-system-dependencies/service-maps/how-use-service-maps
+  - /docs/service-maps-dependencies-new-relic-one
+  - /docs/service-maps-new-relic-one
+  - /node/34551
+  - /nod/34356
+  - /docs/understand-dependencies/understand-system-dependencies/service-maps/service-maps-new-relic-one
+  - /docs/new-relic-one/use-new-relic-one/ui-data/service-maps-new-relic-one
+  - /docs/new-relic-one/use-new-relic-one/ui-data/service-maps/how-use-service-maps
+  - /docs/new-relic-one/ui-data/service-maps/how-use-service-maps
+  - /docs/new-relic-solutions/new-relic-one/ui-data/service-maps/how-use-service-maps
+  - /docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps
+  - /docs/new-relic-one/use-new-relic-one/ui-data/automaps
+  - /docs/new-relic-one/ui-data/automaps
+  - /docs/new-relic-solutions/new-relic-one/ui-data/automaps
+  - /docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context
+---
+
+import SolutionsServiceMaps from 'images/Solutions_screenshot-full_service-map.webp'
+
+import solutionsCriticalHealthStatus from 'images/solutions_icon_critical-health-status.svg'
+
+import solutionsWarningHealthStatus from 'images/solutions_icon_warning-health-status.svg'
+
+import solutionsAlertingHealthStatus from 'images/solutions_icon_alerting-health-status.svg'
+
+import accountsConfiguredHealthStatus from 'images/accounts_icon_configured-health-status.svg'
+
+import solutionsAnomalyHealthStatus from 'images/solutions_icon_anomaly-health-status.webp'
+
+import solutionsMapEntity from 'images/solutions_screenshot-full_map-entity-example.webp'
+
+import solutionsFilterServiceMap from 'images/solutions_screenshot-full_filter-service-map.gif'
+
+import solutionsServiceMapsRightPane from 'images/solutions_screenshot-full_service-maps-right-pane.gif'
+
+import solutionsMapfromthisentityexample from 'images/solutions_screenshot-crop_mapfromthisentityexample.webp'
+
+Service map is a visual representations of your entire architecture from front-end to back-end, providing a comprehensive overview of the interconnected components that make up your system. This map dynamically displays the relationships between your applications, databases, hosts, servers, and out-of-process services (called [web externals]([#externals])). 
+
+<img
+  title="Service map experience"
+alt="A screenshot of the service map experience. The screenshot shows a UI separated by white lines into three tiers. They're ordered vertically with user experience as the top tier, services as the second tier, and infrastructure as the third tier."
+  src={SolutionsServiceMaps}
+/>
+
+<figcaption>
+  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM > (choose an entity) > Service map**: Service map shows your dependencies and how they're performing.
+</figcaption>
+
+## Use service map [#using-service-maps]
+
+<Callout variant="tip">
```

Comment:
Starting the meaty part of the doc with a tip that doesn't _really_ matter breaks the flow a bit too much imo. I'd try to find another location to place this callout.